packages/wdio-browserstack-service/src/insights-handler.ts

```diff
@@ -864,12 +864,15 @@ class _InsightsHandler {
     }
 
     private getIntegrationsObject () {
+        const caps = (this._browser as WebdriverIO.Browser)?.capabilities as WebdriverIO.Capabilities
+        const sessionId = (this._browser as WebdriverIO.Browser)?.sessionId
+
         return {
-            capabilities: this._platformMeta?.caps,
-            session_id: this._platformMeta?.sessionId,
-            browser: this._platformMeta?.browserName,
-            browser_version: this._platformMeta?.browserVersion,
-            platform: this._platformMeta?.platformName,
+            capabilities: caps,
```

Comment:
Do we have existing test cases to cover this?

Reply:
We didn't have existing cases, have added them now.